content/en/docs/installation/uninstall.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Uninstall the notation CLI and its configuration
+description: Uninstall the notation CLI, plugins, and configuration files
+weight: 2
+---
+
+The `notation` CLI uses the following directories for its binaries and configuration files:
+
+- `NOTATION_BIN` is the directory for the `notation` binary.
+- `NOTATION_LIBEXEC` is the directory for other binaries, such as plugins, used by the `notation`.
+- `NOTATION_CONFIG` is the directory for the configuration files, including trust stores and trust policies.
+
+For more details on each directory as well as the location of those directories on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/how-to/directory-structure" >}}).
+
+## Remove the notation CLI
+
+Delete the `NOTATION_BIN` directory to remove the `notation` binary.
+
+## Remove the additional binaries
+
+Delete the `NOTATION_LIBEXEC` directory to remove the additional binaries, including plugins.
+
+## Remove the configuration files
+
+**IMPORTANT:** `notation` automatically creates the `NOTATION_CONFIG` directory if it does not exist. If you remove the `NOTATION_CONFIG` directory, `notation` will recreate it the next time you run a command.
```

Comment:
This is confusing as experience. It seems the first command someone runs creates this directory but I assume it doesn't set any env variables to make the location known - correct? I think we should clarify that _any_ `notation` command does that.

Reply:
updated.